web/src/main/ui/app/components/flows-new/edit-flow/mastering/merging/ui/add-merge-strategy-dialog.component.html

```diff
@@ -43,7 +43,7 @@ <h1 mat-dialog-title>{{getDialogTitle()}}</h1>
           <mat-label>Weight</mat-label>
           <input [class]="'source-weights-weight-' + i" matInput formControlName="weight"/>
         </mat-form-field>
-        <button [class]="'remove-source-weight-btn-' + i" mat-icon-button (click)="onRemoveSourceWeight(i)">
+        <button [id]="'remove-merge-option-source-weight-btn-' + i" mat-icon-button (click)="onRemoveSourceWeight(i)">
```

Comment:
Shouldn't be merge-option

Should be:
<button [id]="'remove-merge-strategy-source-weight-btn-' + i" mat-icon-button (click)="onRemoveSourceWeight(i)">

Reply:
Yikes, that was dumb... fixed.